### PR TITLE
AT_1.01_006 | <Header> Verify Jenkins icon

### PR DESCRIPTION
**Pull request by milladiz84**
https://trello.com/c/Q7lZjxlx/237-at101006-header-verify-jenkins-icon-and-name-icon

### CODE REVIEW

File: cypress/e2e/headerIconVerify.cy.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/// <reference types="cypress"/>
+
+describe('<Header> Head Icon', () => {
```
Comment (LinaAbu1306) [inline cypress/e2e/headerIconVerify.cy.js:3]: Could you please rename your Describe to ' Header | Head Icon "

Comment (milladiz84) [reply cypress/e2e/headerIconVerify.cy.js]: I fixed it, thank you!

**Review by VolhaChudzesa — APPROVED** [review]
It's good!

**Review by milia-menzatova — APPROVED** [review]
Good job!!

**Review by pbeylina — APPROVED** [review]

**Review by TatiChap — APPROVED** [review]
Looks good to me.

**Review by LinaAbu1306 — APPROVED** [review]
Good job!

**Review by SashaFoxx — APPROVED** [review]

**Review by ellayezina — COMMENTED** [review]
well done!

**Pull request merged**